app/controllers/api/v2/config_templates_controller.rb

```diff
@@ -96,13 +98,30 @@ def build_pxe_default
 
       def clone
         @config_template = @config_template.clone
-        load_vars_from_config_template
+        load_vars_from_template
         @config_template.name = params[:config_template][:name]
         process_response @config_template.save
       end
 
+      def resource_name
+        'config_template'
+      end
+
       private
 
+      def type_name_singular
```

Comment:
This overrides the method from the concern `Foreman::Controller::ProvisioningTemplates` so it would be second concern that would have to do aliasing and would be used only in API. If you don't insist I'd like to be explicit here, hopefully V1 will be extracted to some gem and replaced by V2 soon, then the duplicity wouldn't exist anymore.